Source/Core/Core/Boot/Boot.cpp

```diff
@@ -383,7 +383,11 @@ bool CBoot::BootUp(std::unique_ptr<BootParameters> boot)
 {
   SConfig& config = SConfig::GetInstance();
 
-  g_symbolDB.Clear();
+  if (!g_symbolDB.IsEmpty())
+  {
+    g_symbolDB.Clear();
+    UpdateDebugger_MapLoaded();
```

Comment:
💭 This is a different method than in the other two places.

Intentional?

Reply:
UpdateDebugger_MapLoaded just calls Host_NotifyMapLoaded and nothing else. I used it because other code in this file was using it.